packages/react/src/components/LightboxMediaViewer/__stories__/LightboxMediaViewer.stories.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text, object, boolean } from '@storybook/addon-knobs';
+import { DDS_LIGHTBOX_MEDIA_VIEWER } from '../../../internal/FeatureFlags';
+import LightboxMediaViewer from '../LightboxMediaViewer';
+import readme from '../README.md';
+
+import './index.scss';
+
+if (DDS_LIGHTBOX_MEDIA_VIEWER) {
+  storiesOf('LightboxMediaViewer', module)
+    .addDecorator(withKnobs)
+    .addParameters({
+      readme: {
+        sidebar: readme,
+      },
+    })
+    .add('Default', () => {
+      const title = text(
+        'title (required)',
+        'Curabitur malesuada varius mi eu posuere'
+      );
+
+      const copy = text(
+        'copy (required)',
+        `
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. 
+        Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales 
+        nulla quis, consequat libero. Here are 
+        some common categories:
```

Comment:
The way you're using your template literal you're getting this extra space in the input field.

![image](https://user-images.githubusercontent.com/3793636/71212545-2c6cc580-2277-11ea-99a0-60ca50211892.png)